draft-reddy-pquip-pqc-hsm.md

```diff
@@ -445,6 +453,51 @@ current secure boot and update architectures.
 
 Other hybrid techniques, such as cross-linked signatures (where signatures cover each other's values), composite signatures (which combine multiple signatures into a single structured signature), or counter-signatures (where one signature signs over another) introduce more complexity and are not yet typical in resource-constrained firmware workflows.
 
+# Impact of PQC Authentication on Constrained Devices
+
+In constrained environments, devices are often assumed to be clients initiating outbound connections. However,
+many devices also act as servers, enforce local authentication policies, or verify digital signatures on
+firmware, certificates, or commands. These use cases demand inbound authentication capabilities and significantly
+intersect with the challenges of adopting post-quantum cryptography (PQC).
+
+While specific deployment scenarios may differ, the fundamental technical impacts of PQC authentication in constrained devices can be summarized into three main areas:
+
+* Larger Signatures and Certificate Sizes
+
+   Post-quantum signature schemes typically produce much larger public keys and signatures than their classical
+   counterparts. A comparison is provided in [PQC key sizes](#sec-key-sizes).
+
+   These larger artifacts introduce several challenges. For example, certificate chains with PQC public keys
+   require more storage, and trust anchors—particularly for schemes like SLH-DSA—may be too large to embed in
```

Comment:
```suggestion
   require more storage, and trust anchors--particularly for schemes like SLH-DSA--may be too large to embed in
```

I believe emdash isn't allowed in I-D's and RFC's. In any event, in the github 'diff', it looked like a hyphen.